core/iwasm/common/wasm_runtime_common.c

```diff
@@ -2537,8 +2753,8 @@ static VoidFuncPtr invokeNative_Void = (VoidFuncPtr)(uintptr_t)invokeNative;
 bool
 wasm_runtime_invoke_native(WASMExecEnv *exec_env, void *func_ptr,
                            const WASMType *func_type, const char *signature,
-                           void *attachment, uint32 *argv, uint32 argc,
-                           uint32 *argv_ret)
+                           void *attachment, bool aot_call_flag, uint32 *argv,
+                           uint32 argc, uint32 *argv_ret)
```

Comment:
Seems not convert externref index into object pointer in this function (ARM_VFP, THUM_VFP and RISCV32)?

Reply:
Yes. Plan to discuss the return value problem, but seems it has been already mentioned below.